sourced/ml/models/graph_def.py

```diff
@@ -0,0 +1,26 @@
+from modelforge import Model
+
+
+@register_model
+class GraphDef(Model):
+    """
+    Id split model exported a a GraphDef protobuf.
```

Comment:
The name is too generic for such a narrow model. I do not see the Tensorflow serialization here, it should be inside. Also deserialization.

So I would split it into the generic model - please do not name it GraphDef, it collides with the real class name from Tensorflow, - and the concrete model. The latter is out of this particular PRs scope so please do not add it here.

The idsplit model should be self contained. In other words, I expect it to contain the "split" method, or even `__call__`

Reply:
I would recommend making two class: one generic to save/load tensorflow graphs, second for working with keras model as @vmarkovtsev suggested.
Second class should mimic functions from keras - just forward all arguments - it should be easy and convenient to use.
And you could create third class specific for id-splitter only to be optimized for our use cases (convert string to proper input and so on, convert predictions to proper output).

Reply:
Egor, there is no Keras here. It is the lower level GraphDef representation.

Reply:
We are using pure Tensorflow here.